tests/test_enterprise/api/test_views.py

```diff
@@ -4977,3 +4989,54 @@ def test_put_method_not_allowed(self):
         self.assertEqual(response.status_code, 405)
         response = response.json()
         self.assertEqual(response['detail'], 'Method "PUT" not allowed.')
+
+    def test_successful_link(self):
+        """
+        Test `{enterprise_customer_invite_key}/link-user` creates an `EnterpriseCustomerUser`
+        """
+        unlinked_user = factories.UserFactory(
+            username=self.USERNAME,
+            is_active=True,
+            is_staff=False,
+        )
+        unlinked_user.set_password(TEST_PASSWORD)
+        unlinked_user.save()
+
+        client = APIClient()
+        client.login(username=self.USERNAME, password=TEST_PASSWORD)
+
+        response = client.post(
+            settings.TEST_SERVER + reverse(
+                self.ENTERPRISE_CUSTOMER_INVITE_KEY_ENDPOINT_LINK_USER,
+                kwargs={'pk': self.enterprise_customer_3_invite_key.uuid}
+            )
+        )
+        self.assertEqual(response.status_code, 201)
```

Comment:
Should also assert on the API response (e.g., does it return the customers slug in the response)?